staging/src/k8s.io/kubectl/pkg/cmd/apply/applyset.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apply
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+)
+
+var defaultApplySetParentGVR = schema.GroupVersionResource{Version: "v1", Resource: "secrets"}
```

Comment:
If we change this to something different(e.g. configmaps), what will happen when user upgrades their cluster?, will this be a breaking change?

Reply:
Yes, that will be a breaking change, but this is alpha for now, so I'm not worried about such aspect.